Json-Rpc/Config.cs

```diff
@@ -29,6 +39,15 @@ public static void SetPreProcessHandler(PreProcessHandler handler)
         }
 
         /// <summary>
+        /// Sets the the PostProcessing Handler on the default session.
+        /// </summary>
+        /// <param name="handler"></param>
+        public static void SetPostProcessHandler(PostProcessHandler handler)
+        {
+            Handler.DefaultHandler.SetPostProcessHandler(handler);
+        }
```

Comment:
I think we also need one of these functions that will set the handler on a specific sessionId.